templates/order.php

```diff
@@ -54,7 +54,7 @@
                 <!-- Order Status -->
                 <div class="bnomics-order-status-wrapper">
                   <span class="bnomics-order-status-title" ng-show="order.status == -1" ng-cloak ><?=__('To confirm your order, please send the exact amount of <strong>BTC</strong> to the given address', 'blockonomics-bitcoin-payments')?></span>
-                  <span class="warning bnomics-status-warning" ng-show="order.status == -3" ng-cloak><?=__('Payment Expired (Use the browser back button and try again)', 'blockonomics-bitcoin-payments')?></span>
+                  <span class="warning bnomics-status-warning" ng-show="order.status == -3" ng-cloak><?=__('<b>PAYMENT EXPIRED</b> <br /><br /><ul><li><div style="width:100%;display:table;"><span style="width:20%;display:table-cell;"><b>If you already paid :</b></span><span style="display:table-cell;"> Order will automatically confirmed when we detect your payment. You can safely close this window.</span></div></li><li> <b>If you have not already made the payment :</b> <a href="?show_order={{order.address}}&refresh=1">Click here</a> to try again.</li></ul>', 'blockonomics-bitcoin-payments')?></span>
```

Comment:
Can we simplify the UI. Possibly something like this:
![Wordpress Payment Expired Update](https://user-images.githubusercontent.com/39666372/67166382-fe5e2880-f38e-11e9-935f-6377439722fc.png)

Reply:
changed accordingly.